tests/test_embedding.py

```diff
@@ -0,0 +1,63 @@
+import finalfusion
+import pytest
+import numpy
+
+TEST_NORMS = [
+    6.557438373565674,
+    8.83176040649414,
+    6.164413928985596,
+    9.165151596069336,
+    7.4833149909973145,
+    7.211102485656738,
+    7.4833149909973145
+]
+
+
+def test_embeddings_with_norms():
+    embeds = finalfusion.Embeddings(
+        "tests/embeddings.fifu")
+    embeds_dict = dict()
+    with open("tests/embeddings.txt", "r", encoding="utf8")as lines:
```

Comment:
Formatting: `)as lines`

Run autopep8 on the Python file:

https://github.com/hhatto/autopep8

Reply:
sorry... but I just ran autopep8 again on this file (autopep8 --in-place --aggressive --aggressive) , and it didn't change anything.

using autopep8 1.4.4

Reply:
Interesting! seems like a bug in autopep8.